css-multicol-2/Overview.bs

```diff
@@ -360,11 +360,14 @@ The multi-column model</h2>
 
 	The column boxes are ordered
 	in the [=inline base direction=] of the multicol container
-	and arranged into <dfn lt="multi-column line | multi-col line | multicol line">multicol lines</dfn>.
+	and arranged into 
+	<dfn lt="multi-column row | multi-col row | multicol row">multicol rows</dfn>.
+	A multicol row contains at least one <dfn lt="multi-column line | multi-col line | multicol line">multicol line</dfn>,
```

Comment:
A row may also very well just contain a spanner (and no lines), right?

Reply:
Changed this based on another suggestion.